MFVNeutralino/test/histos.py

```diff
@@ -1,135 +1,96 @@
 from JMTucker.Tools.BasicAnalyzer_cfg import *
```

Comment:
If Shaun was relying on this variant of histos.py for all of his studies, should we copy all of this into some new submission script, like btag_trigger_histos.py or something? I don't want us to lose all of that hard work! @shaunhogan @pkotamnives

Reply:
And I wonder if there are other places where we need to do the same. If we are losing functionality by doing the code merge, it's not good! I'm worried that there easily could be things I am missing among the 90k lines and 362 files that have changes. I am actually surprised git didn't have merge conflicts for things like this? Or maybe it did and they were resolved to just one or the other?